### PR TITLE
metrics: update docs

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- Update the part of tidb doc metrics - transaction.
- Update the part of server.
- Update `query_summary`.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5437, https://github.com/pingcap/docs-cn/pull/5412, https://github.com/pingcap/docs-cn/pull/5325
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-tidb-dashboard.md [hunk 1]
```diff
@@ -41,39 +42,39 @@ To understand the key metrics displayed on the TiDB dashboard, check the followi
...
-    - Go Threads: the number of threads on each TiDB instance
-    - Go GC Count: the number of times that Golang GC is executed on each TiDB instance
-    - Go GC CPU Usage: the statistics of CPU used by Golang GC for each TiDB instance
+    - Disconnection Count: the number of clients disconnected to each TiDB instance
     - Events OPM: the statistics of key events, such as "start", "close", "graceful-shutdown","kill", "hang", and so on
-    - Keep Alive OPM: the number of times that the metrics are refreshed every minute on each TiDB instance. It usually needs no attention.
+    - Goroutine Count: the number of Goroutines on each TiDB instance
     - Prepare Statement Count: the number of `Prepare` statements that are executed on each TiDB instance and the total count of them
+    - Keep Alive OPM: the number of times that the metrics are refreshed every minute on each TiDB instance. It usually needs no attention.
+    - Panic And Critical Error: the number that Panic and Critical Errors occur in TiDB
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:50]:
```suggestion
    - Panic And Critical Error: the number of panics and critical errors occurred in TiDB
```

File: grafana-tidb-dashboard.md [hunk 2]
```diff
@@ -41,39 +42,39 @@ To understand the key metrics displayed on the TiDB dashboard, check the followi
...
     - Time Jump Back OPS: the number of times that the operating system rewinds every second on each TiDB instance
-    - Write Binlog Error：the number of times that the binlog write failure occurs every second on each TiDB instance
-    - Handshake Error OPS: the number of times that a handshake error occurs every second on each TiDB instance
     - Get Token Duration: the time cost of getting Token on each connection
+    - Skip Binlog Count: the number of TiDB failing to write into Binlog
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:53]:
```suggestion
    - Skip Binlog Count: the number of binlog write failures in TiDB
```

File: grafana-tidb-dashboard.md [hunk 3]
```diff
@@ -23,11 +23,12 @@ To understand the key metrics displayed on the TiDB dashboard, check the followi
...
+    - QPS: the number of SQL statements executed per second on all TiDB instances, which is counted according to `SELECT`, `INSERT`, `UPDATE`, and other types of statements
+    - CPS By Instance: the command statistics on each TiDB instance, which is classified according to the success or failure of command execution results
     - Failed Query OPM: the statistics of error types (such as syntax errors and primary key conflicts) according to the errors occurred when executing SQL statements per second on each TiDB instance. It contains the module in which the error occurs and the error code
-    - Slow query: the statistics of the processing time of slow queries (the time cost of the entire slow query, the time cost of Coprocessor，and the waiting time for Coprocessor scheduling)
+    - Slow query: the statistics of the processing time of slow queries (the time cost of the entire slow query, the time cost of Coprocessor，and the waiting time for Coprocessor scheduling). Slow queries are classified into internal and general SQL statements.
+    - Connection Idle Duration: duration of idle connection
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:31]:
```suggestion
    - Connection Idle Duration: the duration of idle connections
```

File: grafana-tidb-dashboard.md [hunk 4]
```diff
@@ -23,11 +23,12 @@ To understand the key metrics displayed on the TiDB dashboard, check the followi
...
+    - QPS: the number of SQL statements executed per second on all TiDB instances, which is counted according to `SELECT`, `INSERT`, `UPDATE`, and other types of statements
+    - CPS By Instance: the command statistics on each TiDB instance, which is classified according to the success or failure of command execution results
     - Failed Query OPM: the statistics of error types (such as syntax errors and primary key conflicts) according to the errors occurred when executing SQL statements per second on each TiDB instance. It contains the module in which the error occurs and the error code
-    - Slow query: the statistics of the processing time of slow queries (the time cost of the entire slow query, the time cost of Coprocessor，and the waiting time for Coprocessor scheduling)
+    - Slow query: the statistics of the processing time of slow queries (the time cost of the entire slow query, the time cost of Coprocessor，and the waiting time for Coprocessor scheduling). Slow queries are classified into internal and general SQL statements.
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:30]:
```suggestion
    - Slow query: the statistics of the processing time of slow queries (the time cost of the entire slow query, the time cost of Coprocessor，and the waiting time for Coprocessor scheduling). Slow queries are classified into internal and general SQL statements
```

**Review by TomShawn — APPROVED** [review]
LGTM
@7yyo PTAL, thanks

**Comment by 7yyo** [comment]
LGTM

**Pull request merged**